src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/impl/MtAmazonDynamoDbBySharedTable.java

```diff
@@ -551,6 +591,121 @@ public UpdateItemResult updateItem(UpdateItemRequest updateItemRequest) {
         return getAmazonDynamoDb().updateItem(updateItemRequest);
     }
 
+    @Override
+    public ListBackupsResult listBackups(ListBackupsRequest listBackupsRequest) {
+        if (backupManager.isPresent()) {
+            return backupManager.get().listBackups(listBackupsRequest);
+        } else {
+            throw new ContinuousBackupsUnavailableException("Backups can only be created by configuring a backup "
+                + "managed on an mt-dynamo table builder, see <insert link to backup guide>");
+        }
+    }
+
+    @Override
+    public RestoreTableFromBackupResult restoreTableFromBackup(
+        RestoreTableFromBackupRequest restoreTableFromBackupRequest) {
+        if (backupManager.isPresent()) {
+
+            TenantTableBackupMetadata backupMetadata = ((MtSharedTableBackupManager) backupManager.get())
+                    .getTenantTableBackupFromArn(restoreTableFromBackupRequest.getBackupArn());
```

Comment:
Should we verify here that the current thread tenant context matches the backup arn? (Similar to what we do for streams: https://github.com/salesforce/mt-dynamo/blob/master/src/main/java/com/salesforce/dynamodbv2/mt/mappers/MtAmazonDynamoDbStreamsBase.java#L46)